default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__create_uid
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__create_uid
 msgid "Created by"
 msgstr "Creado por"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,create_date:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__create_date
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__create_date
 msgid "Created on"
 msgstr "Creado en"
 
 #. module: default_warehouse_from_sale_team
-#: field:crm.case.section,default_warehouse:0
+#: model:ir.model,name:default_warehouse_from_sale_team.model_default_picking_type
+msgid "Default Operation Type"
+msgstr "Tipo de Operación por defecto"
+
+#. module: default_warehouse_from_sale_team
+#: model:crm.team,name:default_warehouse_from_sale_team.section_sales_default_team
+msgid "Default Team"
+msgstr "Equipo por defecto"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model,name:default_warehouse_from_sale_team.model_default_warehouse
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__default_warehouse
 #: model:ir.module.category,name:default_warehouse_from_sale_team.default_warehouse_module
 msgid "Default Warehouse"
 msgstr "Almacén por defecto"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,display_name:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__display_name
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__display_name
 msgid "Display Name"
 msgstr "Nombre para mostrar"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,id:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__id
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__id
 msgid "ID"
 msgstr "ID"
 
 #. module: default_warehouse_from_sale_team
-#: help:crm.case.section,default_warehouse:0
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__default_warehouse
 msgid "In this field can be defined a default warehouse for the related users to the sales team."
 msgstr "En este campo se puede definir el almacén por defecto para los usuarios relacionados al equipo de ventas."
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,__last_update:0
+#: model:ir.model,name:default_warehouse_from_sale_team.model_account_invoice
+msgid "Invoice"
+msgstr "Factura"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_stock_id
+msgid "It indicates journal to be used when move line is created withthe warehouse of this sale team"
```

Comment:
- the journal to be used
- a move line
- with the
- sales team

Reply:
Done